pkg/fission-cli/cmd/spec/command.go

```diff
@@ -60,13 +60,22 @@ func Commands() *cobra.Command {
 		Optional: []flag.Flag{flag.SpecDir},
 	})
 
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all Fission resources in the application specification",
+		RunE:  wrapper.Wrapper(List),
+	}
+	wrapper.SetFlags(listCmd, flag.FlagSet{
+		Optional: []flag.Flag{flag.SpecDir},
+	})
+
 	command := &cobra.Command{
 		Use:     "spec",
 		Aliases: []string{"specs"},
 		Short:   "Manage a declarative application specification",
 	}
 
-	command.AddCommand(initCmd, validateCmd, applyCmd, destroyCmd)
+	command.AddCommand(initCmd, validateCmd, applyCmd, destroyCmd, listCmd)
```

Comment:
nit pick: `initCmd, validateCmd, applyCmd, listCmd, destroyCmd`
change the order of command to make list command shows before destroy.
```
# before
Available Commands:
  init        Create an initial declarative application specification
  validate    Validate declarative application specification
  apply       Create, update, or delete resources from application specification
  destroy     Delete all Fission resources in the application specification
  list        List all Fission resources in the application specification

# after:
Available Commands:
  init        Create an initial declarative application specification
  validate    Validate declarative application specification
  apply       Create, update, or delete resources from application specification
  list        List all Fission resources in the application specification
  destroy     Delete all Fission resources in the application specification
```